slides/04_quality.qmd

```diff
@@ -275,48 +275,38 @@ handle exceptions and unexpected situations
 
 ```{r, echo = TRUE, eval = FALSE}
 #| code-line-numbers: false
-getMean("a":"z")
+getMean(c("a", "b", "c"))
```

Comment:
not sure what the intention was here, but `"a":"z"` alone returns a cryptic error message in R, so I don't think this is an indictment of error handling in `getMean()`.